packages/@okta/vuepress-site/guides/sign-into-web-app/configure-packages/springboot/configmid.md

```diff
@@ -5,6 +5,8 @@ okta.oauth2.issuer=https://{yourOktaDomain}/oauth2/default
 okta.oauth2.clientId={clientId}
 okta.oauth2.clientId={clientSecret}
 
-# Customize the callback route path (optional):
+# Customize the callback route path (optional, defaults to '/login/oauth2/code/okta'):
```

Comment:
Do we need to call out the optionality or default value here? For maximum compatibility with other steps in the guide we'd want everyone to always use `/authorization-code/callback` right?

Reply:
you might be right, maybe just drop the "optional" note completely so there is no "well if it's optional, i don't need it" ?